index.js

```diff
@@ -23,42 +25,37 @@ class Client {
         if (options && options.raw) {
           return strValue;
         }
-
         if (!strValue) {
           return null;
         }
-
         let value = strValue;
         try {
           // Try to parse as JSON, if it fails, we throw
           value = JSON.parse(strValue);
         } catch (_err) {
-          throw new SyntaxError(`Failed to parse value of ${key}, try passing a raw option to get the raw value`);
+          value = strValue; //Failed to parse as JSON, return raw STR
```

Comment:
I'm not sure we want to do this without a flag. I think the old behavior is clearer since you know you're either getting back your parsed JSON or an error indicated why you're not getting it back.